developer-tooling/scripts/coverage_runner.py

```
@@ -0,0 +1,85 @@
+
+        # Create necessary directories
+        os.makedirs(static_dir, exist_ok=True)
+        os.makedirs(media_dir, exist_ok=True)
+        os.makedirs(backup_dir, exist_ok=True)
+
+        env = os.environ.copy()
+        env['INVENTREE_STATIC_ROOT'] = static_dir
+        env['INVENTREE_MEDIA_ROOT'] = media_dir
+        env['INVENTREE_DB_ENGINE'] = 'sqlite3'
+        env['INVENTREE_DB_NAME'] = db_path
+        env['INVENTREE_SECRET_KEY'] = 'test_secret_key_12345'
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "INVENTREE_SECRET_KEY"

(S105)